x/bank/handler.go

```diff
@@ -30,14 +31,25 @@ func handleMsgLockedSend(ctx sdk.Context, k Keeper, ak types.AccountKeeper, msg
 	}
 
 	acc = ak.GetAccount(ctx, msg.To)
-	if acc == nil {
-		return nil, sdkerrors.Wrapf(sdkerrors.ErrUnknownAddress, "receiver account %s does not exist", msg.To)
-	}
 
-	// ensure correct account type
-	toAcc, ok := acc.(*vesting.ManualVestingAccount)
-	if !ok {
-		return nil, sdkerrors.Wrapf(sdkerrors.ErrInvalidRequest, "receiver account is not a ManualVestingAccount")
+	var toAcc *vesting.ManualVestingAccount
+	if acc == nil {
+		acc = ak.NewAccountWithAddress(ctx, msg.To)
+		baseAcc := auth.NewBaseAccount(msg.To, sdk.NewCoins(), acc.GetPubKey(), acc.GetAccountNumber(), acc.GetSequence())
+		unlocker, err := sdk.AccAddressFromBech32(msg.Unlocker)
+		if err != nil || msg.Unlocker == "" {
+			return nil, sdkerrors.Wrapf(sdkerrors.ErrInvalidRequest, "invalid unlocker address provided")
+		}
+		toAcc = vesting.NewManualVestingAccount(baseAcc, sdk.NewCoins(), unlocker)
+	} else {
+		var ok bool
+		toAcc, ok = acc.(*vesting.ManualVestingAccount)
+		if !ok {
```

Comment:
Should we check if the unlocker is the same as the one in the account? Ignoring it silently is not a good idea.